pkg/pb/metainfo.pb.go

```diff
@@ -3,15 +3,17 @@
 
```

Comment:
**Rational for code Changes:**
Changes due to protobuf changes and corresponding code generation tool